### PR TITLE
Add Lenovo Tab 3 8 850M manifest

### PR DESCRIPTION
**Pull request by aL31cS**

### CODE REVIEW

File: manifests/lenovo_tab3.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<remote name="mygit"
```
Comment (Vince1171) [inline manifests/lenovo_tab3.xml]: can you please change the remote name to `aL31cS`? `mygit` is too generic

**Review by Vince1171 — CHANGES_REQUESTED** [review]

File: manifests/lenovo_tab3.xml (hunk 1 above)
Comment (ghost) [reply manifests/lenovo_tab3.xml]: Ok

**Comment by Vince1171** [comment]
can you regroup all commits in one and forcepush? (named `Add Lenovo Tab 3 850M manifest`)

**Comment by ghost** [comment]
> can you regroup all commits in one and forcepush? (named `Add Lenovo Tab 3 850M manifest`)

Done

**Comment by aL31cS** [comment]
Yes

mie., 10 feb. 2021, 18:45 Vince1171 <notifications@github.com> a scris:

**Comment by Vince1171** [comment]
I've removed the extra commits ;-)

**Pull request merged**